src/seedu/addressbook/Main.java

```diff
@@ -83,7 +83,7 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
+            command = new Parser(ui).parseCommand(userCommandText);
```

Comment:
Probably seems like a hack, as this change affects all commands here (usually not recommended). 
This means that all the commands may need to be tested (with possibly new test cases).